arm-network/2017-03-01/swagger/virtualNetwork.json

```diff
@@ -1047,24 +1048,29 @@
         "currentValue": {
           "type": "number",
           "format": "double",
-          "description": "Current usage value."
+          "readOnly": true,
+          "description": "Indicates number of IPs used from the Subnet."
```

Comment:
Can there be 10.5 IPs in used? I am asking because we have specified `"format": "double"`